ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -0,0 +1,44 @@
+package siteConfig
+
+import (
+	"log"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+// Add ztp deploy annotation to all siteconfig generated CRs
+// A dynamic value might be added later
+func addZTPAnnotation(clusterCRs []interface{}) ([]interface{}, error) {
+
+	for _, v := range clusterCRs {
+		cr := v.(map[string]interface{})
+		metadata := cr["metadata"].(map[string]interface{})
+		annotations, _ := metadata["annotations"].(map[string]interface{})
```

Comment:
Do we need the `, _` if we're just trying to get the value?

Reply:
No, it is not needed.